File: workoutpixel/src/test/java/ch/karimattia/workoutpixel/CommonFunctionsTest.java
package ch.karimattia.workoutpixel;

import ch.karimattia.workoutpixel.core.CommonFunctions;

import org.junit.Assert;
import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class CommonFunctionsTest {

    @Test
    public void next3AmTest() {

        Assert.assertEquals(CommonFunctions.timeBeautiful(CommonFunctions.next3Am()), "03:00");
        assertEquals(CommonFunctions.timeBeautiful(CommonFunctions.today3Am()), "03:00");
        assertEquals(CommonFunctions.timeBeautiful(CommonFunctions.last3Am()), "03:00");
    }
}

File: workoutpixel/src/test/java/ch/karimattia/workoutpixel/CommonFunctionsTest.kt
package ch.karimattia.workoutpixel

import ch.karimattia.workoutpixel.core.timeBeautiful
import ch.karimattia.workoutpixel.core.next3Am
import ch.karimattia.workoutpixel.core.today3Am
import ch.karimattia.workoutpixel.core.last3Am
import org.junit.Assert
import org.junit.Test

class CommonFunctionsTest {
	@Test
	fun next3AmTest() {
		Assert.assertEquals(timeBeautiful(next3Am()), "03:00")
		Assert.assertEquals(timeBeautiful(today3Am()), "03:00")
		Assert.assertEquals(timeBeautiful(last3Am()), "03:00")
	}
}